ocfweb/docs/docs/services/web/jekyll.md

```diff
@@ -0,0 +1,67 @@
+[[!meta title="Jekyll"]]
+
+[Jekyll](https://jekyllrb.com) is a popular static website generator.
+
+## Set up
+
+1. Go to our [web-based SSH client](https://ssh.ocf.berkeley.edu/) and sign in
+   with your username and password. Or, at terminal, [SSH](https://www.ocf.berkeley.edu/docs/services/shell/) into tsuanami.ocf.berkeley.edu
+
+2. Set up [rbenv](https://github.com/rbenv/rbenv):
+```shell
+  echo 'export PATH="~/.rbenv/bin:$PATH"' >> ~/.bash_profile
+  echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
```

Comment:
We should also make sure that `rbenv` is installed on Debian Stretch machines, so when tsunami is upgraded to that OS version, it will still have `rbenv` installed. We can do this using Puppet; I can show you how at my Friday 3pm-or-after staff hours.